artifacttypes/http%3A%2F%2Fopentosca.org%2Fartifacttypes/PythonArchiveArtifact/ArtifactType.tosca

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Definitions targetNamespace="http://opentosca.org/artifacttypes" id="artifactTypes-PythonArchiveArtifact" xmlns="http://docs.oasis-open.org/tosca/ns/2011/12" xmlns:yml="http://docs.oasis-open.org/tosca/ns/simple/yaml/1.3" xmlns:selfservice="http://www.eclipse.org/winery/model/selfservice" xmlns:winery="http://www.opentosca.org/winery/extensions/tosca/2013/02/12" xmlns:testwineryopentoscaorg="http://test.winery.opentosca.org">
+    <ArtifactType name="PythonArchiveArtifact" abstract="no" final="no" targetNamespace="http://opentosca.org/artifacttypes"/>
```

Comment:
Maybe inherit from the Archive artifact type?

Reply:
added